README.md

```diff
@@ -57,6 +56,19 @@ The services rely on a configuration file in `/etc/irods/` which dictates which
 
 Once the REST API is running install nginx and then copy `/etc/irods/irods-client-rest-cpp-reverse-proxy.conf.template` to `/etc/nginx/sites-available/irods-client-rest-cpp-reverse-proxy.conf` and then symbolically link it to `/etc/nginx/sites-enabled/irods-client-rest-cpp-reverse-proxy.conf`.   Nginx will then need to be restarted with `sudo service nginx restart`, or however your specific platform manages services.
 
+## Enabling logging via Rsyslog
+The REST API uses rsyslog for logging. To enable, copy `packaging/00-irods-client-rest-cpp.conf` into `/etc/rsyslog.d` and restart rsyslog.
+
+The log file will be located at `/var/log/irods/irods_client_rest_cpp.log`.
+
+The log level for each endpoint can be adjusted by modifying the `"log_level"` option in `/etc/irods/irods_client_rest_cpp.json`. The following values are supported:
+    - "trace"
+    - "debug"
+    - "info"
+    - "warn"
+    - "error"
+    - "critical"
+
```

Comment:
@trel Thoughts?

Reply:
seems good/sufficient.

is `packaging/00-irods-client-rest-cpp.conf` created/populated via the deb/rpm package?    need to consider a no-source deployment / ease-of-use.

maybe we have a template file and place it in `/etc/irods/00-irods-client-rest-cpp.conf.template`?   and then document the `cp` command to get it into `/etc/rsyslog.d/`?

Reply:
That works. I didn't want to have the package install a file into rsyslog.d or logrotate.d given those apps aren't always available.

Also, nginx didn't install an rsyslog config file. It did install a logrotate config file though. I'm not sure if we want to do that or not. The main iRODS server will install the config files for a fresh iRODS deployment, but not on upgrade. We require the admin to run a script if doing an upgrade.

Reply:
done